.ci_scripts/push_changes

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+if [ -n "$GH_TOKEN" ]; then
```

Comment:
We could add in the neat updating/rebasing tricks at this point. However, I don't think it should be necessary. Part of the move here is to ensure that this repo is updated at regular well-spaced intervals. Am hoping that this seldom sees any developmental changes. Ideally the only changes will be moving the scripts out of here and into something else that is used as a dependency.